android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/auth/AuthManager.java

```diff
@@ -0,0 +1,341 @@
+                    public void call(UserSessionInfo userSessionInfo) {
+                        authManagerDelegateProtocol.setEmail(email);
+                        authManagerDelegateProtocol.setPassword(password);
+                        authManagerDelegateProtocol.setUserSessionInfo(userSessionInfo);
+                    }
+                }).doOnError(new Action1<Throwable>() {
+                    @Override
+                    public void call(Throwable throwable) {
+                        if (throwable instanceof NotAuthenticatedException) {
+                            authManagerDelegateProtocol.setPassword(null);
+                        }
+                    }
+                });
+    }
+
+    /**
+     * On success, clears the participant's email, password and session from storage
+     *
+     * @return notifies of completion or error
+     */
+    @NonNull
+    public Completable signOut() {
+        return RxUtils.toBodySingle(authenticationApi.signOut())
+                .doOnSuccess(new Action1<Message>() {
+                    @Override
+                    public void call(Message message) {
+                        authManagerDelegateProtocol.setEmail(null);
+                        authManagerDelegateProtocol.setPassword(null);
+                        authManagerDelegateProtocol.setUserSessionInfo(null);
+                    }
+                }).toCompletable();
+    }
+
+    /**
+     * An email will be sent if an account is registered with the configured study.
+     *
+     * @param email participant's email address
+     * @return notifies of completion or error
+     */
+    @NonNull
+    public Completable requestPasswordResetForEmail(@NonNull String email) {
+        checkNotNull(email);
+
+        return RxUtils.toBodySingle(authenticationApi.requestResetPassword(new Email().study
+                (config.getStudyId()).email(email))).toCompletable();
+    }
+
+    /**
+     * On success, the participant's new password will be stored.
+     *
+     * @param password   new password to use
+     * @param resetToken one-time use password reset token, normally emailed to a user's account
+     * @return notifies of completion or error
+     */
+    @NonNull
+    public Completable resetPasswordToNewPassword(@NonNull final String password, @NonNull
+            String resetToken) {
+        checkNotNull(password);
+        checkNotNull(resetToken);
+
+        return RxUtils.toBodySingle(
+                authenticationApi.resetPassword(
+                        new PasswordReset()
+                                .password(password)
+                                .sptoken(resetToken)
+                ))
+                .doOnSuccess(new Action1<Message>() {
+                    @Override
+                    public void call(Message message) {
+                        authManagerDelegateProtocol.setPassword(password);
+                    }
+                }).toCompletable();
+    }
```

Comment:
I don't think you'll ever be in a position to call this. The request reset password call triggers an email and that has a link back to a web page on BridgePF where you can change the password. If this call is in the SDK, I'm not sure how we'd use it, even though this API is part of the Bridge API for that web page.